### DISCUSSION TITLE
Is it possible to use it with podman instead of docker

### DISCUSSION THREAD

**Discussion by yasirmturk** [kool-dev/kool#438] [Ideas]
**Describe the feature**

podman is a replacement for docker 

**Backward compatibility concerns**

Would this be BC?

**Use cases**

Situations and audience for this feature.

**Examples**

If applicable, add pseudo-commands examples to help explain your idea.

**Comment by danielsuguimoto** [comment]
Hey @yasirmturk! Thanks for your interest in our tool! Sorry, but it isn't possible at the moment. Kool uses Docker CLI under the hood. We have plans to drop some dependencies, docker compose for example...but nothing concrete and short tem

**Reply by yasirmturk** [reply]
Hi @danielsuguimoto is there any update on this? as docker has moved to commercial licensing, there might be more requirement to adopt `podman`
